packages/twenty-front/src/modules/object-record/record-store/contexts/RecordFieldValuesSelectorContext.tsx

```diff
@@ -0,0 +1,16 @@
+import { recordFieldsStoreFamilySelector } from '@/object-record/record-store/states/selectors/recordFieldsStoreFamilySelector';
+import { useRecoilValue } from 'recoil';
+
+export const useRecordFieldValues = <T,>(
```

Comment:
I'm a bit lost, what's the purpose of this util and state?

Reply:
got it but we can remove this hook, let's use useRecoilValue directly when we need

Reply:
no pb. I was trying to follow the way it was done for RELATION.